CONTRIBUTING.md

```diff
@@ -0,0 +1,223 @@
+
+## Getting started
+
+The steps below will give you a general idea of how to prepare your local environment for nodejs loaders development.
+
+1. Click the fork button in the top right to clone the [Node.js Website Repository](https://github.com/nodejs-loaders/nodejs-loaders/fork)
+
+2. Clone your fork using SSH, GitHub CLI, or HTTPS.
+```bash
+git clone git@github.com:<YOUR_GITHUB_USERNAME>/nodejs-loaders.git # SSH
+git clone https://github.com/<YOUR_GITHUB_USERNAME>/nodejs-loaders.git # HTTPS
+gh repo clone <YOUR_GITHUB_USERNAME>/nodejs-loaders.org # GitHub CLI
+```
+
+3. Change into the directory.
+```bash
+cd nodejs-loaders
+```
+
+4. Create a remote to keep your fork and local clone up-to-date.
+```bash
+git remote add upstream git@github.com:nodejs-loaders/nodejs-loaders.git # SSH
+git remote add upstream https://github.com/nodejs-loaders/nodejs-loaders.git # HTTPS
+gh repo sync nodejs-loaders/nodejs-loaders # GitHub CLI
+```
+
+5. Create a new branch for your work.
+```bash
+git checkout -b my-new-feature
+```
+
+6. Install the dependencies.
+```bash
+npm install
+```
+
+7. Run the tests
+```bash
+node --run test
+```
+
+8. Run lint and formatting
+```bash
+node --run pre-commit
+```
+
+9. Let's make some changes!
+
+10. Keep your fork up-to-date with the upstream repository.
+```bash
+git fetch upstream
+git merge upstream/main
+```
+
+11. Once you're ready, push your changes to your fork.
+```bash
+git add .
+git commit -m "describe your changes"
+git push -u origin name-of-your-branch
+```
+
+## Commit Message Guidelines
+
+- Commit messages must include a "type" as described on [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/)
+- Commit messages **must** start with a capital letter
+- Commit messages **must not** end with a period `.`
+
+## Pull Request Guidelines
+
+Before merging a pull request, make sure the following requirements are met:
+
+- The pull request has a descriptive title and follows the commit message guidelines
+- An approval is valid if there have been no major changes since it was granted.
+- 24 hours after approval and no objections, the pull request can be merged
+- All tests pass
+
+## Introduce a New Loader
+
+If you want to introduce a new loader, please follow the steps below:
+
+- Open an issue to discuss the new loader
+  - Describe the use case for the new loader
+	- What problem does it solve?
+	- What are the benefits of the new loader?
+	- Which dependencies will be required?
+- Once the issue is approved, make changes to the codebase
+- Open a pull request
+- Once the pull request is approved, the new loader will be merged
+- Let's realease it.
+
+### Create a New Loader
+
+1. Create a new directory in the `packages` directory with the name of the loader
+2. Create the package.json file
+```json
+{
+	"version": "1.0.0",
+	"name": "@nodejs-loaders/your-loader",
+	"type": "module",
+	"author": "Your Name",
+	"maintainers": [
+		"Jacob Smith",
+		"Augustin Mauroy"
```

Comment:
We can make it alphabetical order so there's no inherent priority

```suggestion
		"Augustin Mauroy",
		"Jacob Smith"
```

Reply:
You should assume that I'm the best 😂